### PR TITLE
Sync subscriptions

### PR DESCRIPTION
**Pull request by tspenov**

### CODE REVIEW

File: config/scheduler_config.exs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    ],
+    trending_words_sonar_signal: [
+      schedule: "5-59/5 * * * *",
+      task: {Sanbase.Signal.Scheduler, :run_signal, [Trigger.TrendingWordsTriggerSettings]}
+    ],
+    eth_wallet_signal: [
+      schedule: "5-59/5 * * * *",
+      task: {Sanbase.Signal.Scheduler, :run_signal, [Trigger.EthWalletTriggerSettings]}
+    ],
+    sync_stripe_subscriptions: [
```
Comment (IvanIvanoff) [inline config/scheduler_config.exs]: This will run in the `signals` pod, is that ok?

Comment (IvanIvanoff) [reply config/scheduler_config.exs]: Let's separate the schedulers into `Sanbase.Scrapers.Scheduler` and `Sanbase.Signals.Scheduler` and start the appropriate ones in the supervision tree. Also we need to change the schedulers  in the config

**Comment by tspenov** [comment]
devops PR: https://github.com/santiment/devops/pull/962

File: config/scheduler_config.exs (hunk 1 above)
Comment (tspenov) [reply config/scheduler_config.exs]: split the signals and scrapers and put in scrapers, though it doesn't matter for this one cause it needs only ecto

**Pull request merged**